NodeBase/entry_point.sh

```diff
@@ -25,6 +25,10 @@ if [ ! -z "$REMOTE_HOST" ]; then
   REMOTE_HOST_PARAM="-remoteHost $REMOTE_HOST"
 fi
 
+if [ ! -z "$FIREFOX_VERSION" ]; then
+  sudo ln -fs /opt/firefox/${FIREFOX_VERSION}/firefox /usr/bin/firefox
+fi
+
```

Comment:
i'm aware that it's a no-op if you, of course, do not specify `FIREFOX_VERSION`, but still interesting to hear your reasoning :)